WinUIGallery/ControlPages/TitleBarPage.xaml.cs

```diff
@@ -139,5 +165,37 @@ private void defaultTitleBar_Click(object sender, RoutedEventArgs e)
         {
             SetTitleBar(null);
         }
+
+        public OverlappedPresenter Presenter
+        {
+            get => AppWindow.GetFromWindowId(Win32Interop.GetWindowIdFromWindow(WinRT.Interop.WindowNative.GetWindowHandle(WindowHelper.GetWindowForElement(this as UIElement)))).Presenter as OverlappedPresenter;
+        }
+
+        private bool _hasBorder = true;
+        private bool _hasTitleBar = true;
+        public bool HasBorder
+        {
+            get => _hasBorder;
+            set
+            {
+                _hasBorder = value;
+                Presenter.SetBorderAndTitleBar(value, _hasTitleBar);
+            }
+        }
+        public bool HasTitleBar
+        {
+            get=> _hasTitleBar;
+            set
+            {
+                _hasTitleBar = value;
+                if (_hasTitleBar)
+                {
+                    HasBorder = true;
+                    OnPropertyChanged("HasBorder");
+                }
+                Presenter.SetBorderAndTitleBar(_hasBorder, value);
+                OnPropertyChanged("HasTitleBar");
+            }
+        }
```

Comment:
I would prefer having CheckBox.Checked handlers for this instead of properties since the handlers are a lot easier to understand.